README.md

```
@@ -235,6 +243,18 @@ https://www.npmjs.com/package/hili-lipsum
 
 <br>
 
+## References
+
+#### Genshin Impact Fandom Wiki
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)